redi-connect-front/src/components/molecules/Editable.tsx

```diff
@@ -1,43 +1,49 @@
 import React, { useState } from 'react'
-import { Element } from 'react-bulma-components'
+import { Heading } from 'react-bulma-components'
+import classnames from 'classnames'
 import './Editable.scss'
 
 interface Props {
   title: string
   onSave: Function
   read: React.ReactNode
   children: React.ReactNode
+  className?: string
+  savePossible?: boolean
 }
 
 function Editable (props: Props) {
```

Comment:
Niiiice.

If you want to make this even nicer, consider the "Compount Components" pattern by Kent C Dodds. Check the `Usage` component starting line 43 in this codesandbox to get a quick idea: https://codesandbox.io/s/github/kentcdodds/advanced-react-patterns-v2?file=/src/exercises-final/03.js

Reply:
interesting approach (will need some time to wrap my head arround), will follow up on this in the future.

Reply:
Cool. Here's an explanation on his blog: https://kentcdodds.com/blog/compound-components-with-react-hooks/